build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Check if the script is being run from the root folder (check for docker/Dockerfile)
+if [ ! -f "docker/Dockerfile" ]; then
+    echo "Error: Dockerfile not found in docker/ folder. Please run this script from the root of the repository."
+    exit 1
+fi
+
+# Build the Docker image
+echo "Building the Docker image..."
+docker build -f docker/Dockerfile -t shunya-dev .
+
+echo "Docker image built successfully."
```

Comment:
It seems like this line echoes a success even if the previous steps fail, could be misleading. I forced a build failure to test this:
![Screenshot from 2024-10-19 12-32-18](https://github.com/user-attachments/assets/d670430e-a0e9-4484-ba33-21095da5ae41)

Reply:
I think you can add a `set -e` at the beginning of the script so that it exits immediately after an error or add a if/else logic to declare success or failure

Reply:
I have added another line, please check

Reply:
Looks good to me